troupon/troupon/urls.py

```diff
@@ -5,8 +5,9 @@
 from account import views
 
 urlpatterns = [
-    url(r'^auth/', include('account.urls')),
+	url(r'^auth/', include('account.urls')),
```

Comment:
Okay so while rebasing with develop, I noticed that some views (e.g. /signin/) used the /auth/ as the base URL. There are also tests written to test that route and they all failed when I removed it. I'm not exactly sure if it's okay to edit/remove code written by somebody else while collaborating especially when it has been merged to develop and the source branch has been deleted. So I left it for 'compatibility' though I'm not exactly using it. What say you? I remove it?